host-provision.yml

```diff
@@ -61,7 +61,7 @@
   vars:
     ansible_ssh_private_key_file: "{{ control_node_key_file }}"
     #ansible_ssh_extra_args can be input as a command line or the following reasonable default may be used:    
-    ansible_ssh_extra_args: '-o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o IdentityAgent=no '
+    ansible_ssh_extra_args: ' -o IdentityAgent=no '
```

Comment:
Nit: not sure the leading and trailing spaces are necessary

Reply:
It is true they are required, but I left the spaces as a placeholder/reminder, if in the future we get the `ansible_ssh_extra_args` from the command line and then stitch together a complex option (with many `-o`s), for example (this is a real-life example!):
```
    ansible_ssh_extra_args: '-o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o IdentityAgent=no -o ProxyCommand="ssh -W %h:%p -q jcnarasimhan@control-node.mfild.com"'
```

Will remove the spaces.

Reply:
Done